ledger/heavy/executor/jetkeeper_test.go

```diff
@@ -31,7 +33,126 @@ import (
 	"github.com/insolar/insolar/internal/ledger/store"
 )
 
+func TestJetInfoIsConfirmed_OneDropOneHot(t *testing.T) {
```

Comment:
Can we try blackbox testing?
In our case test public functions is very easy and doesn't require compicated bootstrap. It will yield cleaner tests that are easy to maintain.